great_expectations/expectations/expectation.py

```diff
@@ -2473,7 +2601,7 @@ def _validate(
 
 class MulticolumnMapExpectation(TableExpectation, ABC):
     map_metric = None
-    domain_keys = (
+    domain_keys = (  # type: ignore[assignment]
```

Comment:
same question as above

Reply:
See my write-up in Confluence or we can discuss offline.